src/strategies/api.ts

```diff
@@ -1,5 +1,34 @@
-// TODO: xhr call to private rest api goes here
+import InstagramClient from 'instagram-private-api';
```

Comment:
They actually don't ship types :( so everything is any... We should at least create internal interfaces and map their result to us.